test/functional/page_objects/visualize_page.js

```diff
@@ -481,13 +477,12 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
       // So to modify a metric or aggregation tests need to keep track of the
       // order they are added.
       await this.toggleOpenEditor(index);
-      const aggSelect = await find
-        .byCssSelector(`#visAggEditorParams${index} div [data-test-subj="visEditorAggSelect"] div span[aria-label="Select box activate"]`);
-      // open agg selection list
-      await aggSelect.click();
+
       // select our agg
-      const aggItem = await find.byCssSelector(`[data-test-subj="${agg}"]`);
-      await aggItem.click();
+      const aggSelect = await find
+        .byCssSelector(`#visAggEditorParams${index} div [data-test-subj="defaultEditorAggSelect"]`);
+      await comboBox.setElement(aggSelect, agg);
+
       const fieldSelect = await find
         .byCssSelector(`#visAggEditorParams${index} > [agg-param="agg.type.params[0]"] > div > div > div.ui-select-match > span`);
```

Comment:
I'm guessing this will no longer work anymore as well?

Reply:
This still works because `Field` select control hasn't been migrated yet.

![image](https://user-images.githubusercontent.com/11524637/54670915-ca273e80-4b05-11e9-89ff-e2ce048c68cc.png)